src/register-org/models/registrationdata.model.ts

```diff
@@ -9,4 +9,10 @@ export type RegistrationData = {
   hasPBA: boolean;
   contactDetails: ContactDetails;
   hasRegisteredWithRegulator: boolean;
+  companyHouseNumber?: string;
+  address: string;
+  organisationType: string;
+  organisatinonNumber?: string;
+  regulatoryOrgType: string;
+  reulatorRegisteredWith: string;
```

Comment:
@DavidJayakumar - looks like a typo here; perhaps you meant `regulatorRegisteredWith`?